src/coprocessor/Client.cc

```diff
@@ -726,10 +727,10 @@ void ResponseIter::handleTask(const CopTask & task)
     {
         if (is_cancelled || meet_error)
             return;
+        const auto & current_task = remain_tasks[idx];
+        auto & bo = bo_maps.try_emplace(current_task.region_id.id, kv::copNextMaxBackoff).first->second;
```

Comment:
Is this line of change related to the log format issue?

Reply:
I found it is hard to locating which region meet the lock error, and which lock_version is prevent the request being handle.
So I move these two lines out of the `try ... catch ...` scope, in order to log down the region_id when error happens in line 744.

Reply:
I see, thank you.